addons/gamification/data/goal_base.xml

```diff
@@ -33,7 +33,7 @@
             <field name="computation_mode">count</field>
             <field name="display_mode">boolean</field>
             <field name="model_id" ref="base.model_res_company"/>
-            <field name="domain">[('user_ids', 'in', [user.id]),('logo', '!=', False)]</field>
```

Comment:
This needs to be adapted to be valid when there is no actual image (ignoring placeholder).
We still want to encourage people to set their company logo, even if there is a placeholder.